### PR TITLE
Change 2019-10-preview to 2019-10-01-preview

### PR DESCRIPTION
**Pull request by jqhuangonearth**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/search/resource-manager/readme.md [hunk 1]
```diff
@@ -48,10 +48,10 @@ input-file:
 - Microsoft.Search/stable/2015-02-28/search.json
 ```
 
-### Tag: package-2019-10-preview
+### Tag: package-2019-10-01-preview
```
Comment (abhi1509) [inline specification/search/resource-manager/readme.md]: should package-2019-10-01-preview be package-2019-10-01-**P**review

**Review by abhi1509 — COMMENTED** [review]
Are you also renaming the folder with this change?

**Comment by abhi1509** [comment]
#sign-off

**Review by abhi1509 — APPROVED** [review]

**Review by bitsforthought — CHANGES_REQUESTED** [review]
There is no need for this change that I can see. It is standard practice to group tags by year and month.

**Comment by huangbolun** [comment]
It is standard practice to group tags by year and month. So the fix is not necessary.

**Pull request closed without merging**